docs/introduction/scope.md

```diff
@@ -0,0 +1,29 @@
+# Scope
```

Comment:
Can we rename it to *Scoping* ? To avoid confusion that we have something with `type: vf:Scope`. But we can use references to anything to scope events and commitments.